alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -123,6 +124,7 @@ class MonitorRunner(
 
     @Volatile private var destinationSettings = loadDestinationSettings(settings)
     @Volatile private var destinationContextFactory = DestinationContextFactory(client, xContentRegistry, destinationSettings)
+    @Volatile private var awsSettings = AWSSettings.parse(settings)
```

Comment:
The way this is currently, the `access/secret` key settings will be loaded from the keystore only upon bootstrapping of ES. If the user wanted to change those settings, they would need to restart the cluster which isn't ideal.

You might want to extend `reloadDestinationSettings` to support updating these settings using the reload secure settings API (`POST _nodes/reload_secure_settings`)

Reply:
thanks for pointing that out, fixed